UPDATE.md

```diff
@@ -65,6 +67,10 @@ Beta 2, then from Beta 2 to Beta 3, in that order.
   actively using Page Manager, you must add it to your project as an explicit
   dependency in order to continue to using it. Otherwise, you should uninstall
   it before updating to the next version of Lightning.
+* Lightning now supports exposing all Drupal entities as JSON, in the standard
+  JSON API format. To enable this feature, install the OpenAPI, OpenAPI Redoc
+  Documentation, and JSON API modules. Then install the Content API module from
+  the Lightning package.
```

Comment:
The reason they are not dependencies is because there was no need for them to be. They are all installed by the Lightning profile, but Content API is not actually doing anything which outright *requires* any of them. To keep things relatively cherry-pickable (i.e., for PS), I prefer to avoid defining hard dependencies when they're not needed.